### PR TITLE
Standardize Font Sizes

### LINKED ISSUE #83: Font Size Adjustments
## Description

Currently the font sizes are too big across the site, we want to make font sizes smaller. Confirm with @jocelynxu01 and designer on this one. 

## To-Dos

- [ ] Check through font (work with Joos and Mandy on this one)
- [ ] Check through font sizes

### PR DESCRIPTION
**Pull request by anthonycruzmacedo**
## Summary
- Closes #83 
- Standardize fonts across the site to fit header-1, header-2, header-3, body-bold, body-normal, body-small as defined in Figma and below

style | font-family | font-weight | font-size 
--- | --- | --- | --- |
header-1 | Klima | 647 | 32 
header-2 | Klima | 559 | 28 
header-3 | Klima | 500| 22 
body-bold | Inter | 500 | 16 
body-normal | Inter | 400 | 16 | 19
body-small | Inter | 400 | 12 | 15

## Changes
- add font style options (header-1,header-2,header-3,body-bold,body-normal,body-small) to styles/globals.css
- multiple changes to components and pages (for specific uses)

### CODE REVIEW

**Review by zorazrr — CHANGES_REQUESTED** [review]
Nice work!! The site is looking much cleaner :D. 
Some naming and styling cleanups!

File: src/styles/globals.css [hunk 1]
```diff
@@ -9,6 +9,32 @@
...
+@layer components {
+  .t1 {
+    @apply font-klima text-[32px] font-[647] leading-9;
+  }
+
+  .t2 {
+    @apply font-klima text-[28px] font-[559] leading-8;
+  }
+
+  .t3 {
```
[the remark this answers is not in the document]
Comment (anthonycruzmacedo) [reply src/styles/globals.css]: replaced t with header-, renamed bb to body-bold, r1/r2 to body-normal/body-small

File: src/styles/globals.css [hunk 2]
```diff
@@ -9,6 +9,32 @@
...
+    @apply font-klima text-[22px] font-[559] leading-[25px];
+  }
+
+  .bb {
+    @apply font-inter text-base font-medium leading-[19px];
+  }
+
+  .r1 {
```
[the remark this answers is not in the document]
Comment (anthonycruzmacedo) [reply src/styles/globals.css]: Placed in body-bold in _app.tsx as default font option

File: src/components/Buttons/BackButton.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ const BackButton: FC<BackButtonProps> = ({
       onClick={handleClick}
       className={clsx(
         clsx(
-          'flex w-fit place-content-center items-center justify-center truncate p-4 font-inter text-lg',
+          'bb flex w-fit place-content-center items-center justify-center truncate p-4 font-inter',
```
[the remark this answers is not in the document]
Comment (anthonycruzmacedo) [reply src/components/Buttons/BackButton.tsx]: removed repetitive uses of font-x, font-[x], text-x, text-[x], some remain to override default font styles in specific cases

File: src/styles/globals.css [hunk 4]
```diff
@@ -9,6 +9,32 @@
...
+    @apply font-inter text-base font-normal leading-[19px];
```
[the remark this answers is not in the document]
Comment (anthonycruzmacedo) [reply src/styles/globals.css]: removed line height styles from globals.css font options

File: src/components/Selects/SearchBar.tsx [hunk 5]
```diff
@@ -26,11 +26,11 @@ const SearchBar: FC<SearchBarProps> = ({ setCompanySearchQuery }) => {
             id="companyName"
             name="companyName"
             placeholder="Search by Company Name"
-            className="grow rounded pl-14"
+            className="bb grow rounded pl-14"
           />
           <button
             type="submit"
-            className="underline-offset-6 ml-6 flex items-center justify-center rounded-full bg-black px-6 text-white underline"
+            className="bb underline-offset-6 ml-6 flex items-center justify-center rounded-full bg-black py-1 px-4 text-white underline"
```
[the remark this answers is not in the document]
Comment (anthonycruzmacedo) [reply src/components/Selects/SearchBar.tsx]: removed all instances of body-bold

File: src/components/Accordions/CompanyDetailsAccordion.tsx [hunk 6]
```diff
@@ -29,7 +29,7 @@ const CompanyDetailsAccordion: FC<CompanyDetailsAccordionProps> = ({
   return (
     <div
       className={clsx(
-        'rounded-[10px] border-[3px] border-cobalt bg-lightBlue px-10 py-5',
+        ' rounded-[10px] border-[3px] border-cobalt bg-lightBlue px-10 py-5',
```
Comment (zorazrr) [inline src/components/Accordions/CompanyDetailsAccordion.tsx:32]:
```suggestion
        'rounded-[10px] border-[3px] border-cobalt bg-lightBlue px-10 py-5',
```

File: src/components/Buttons/BackButton.tsx [hunk 7]
```diff
@@ -27,7 +27,7 @@ const BackButton: FC<BackButtonProps> = ({
       onClick={handleClick}
       className={clsx(
         clsx(
-          'flex w-fit place-content-center items-center justify-center truncate p-4 font-inter text-lg',
+          ' flex w-fit place-content-center items-center justify-center truncate p-4',
```
Comment (zorazrr) [inline src/components/Buttons/BackButton.tsx:30]:
```suggestion
          'flex w-fit place-content-center items-center justify-center truncate p-4',
```

File: src/components/Buttons/LoadMoreButton.tsx [hunk 8]
```diff
@@ -10,8 +10,7 @@ const LoadMoreButton: FC<LoadMoreButtonProps> = ({ onClick, disabled }) => {
   return (
     <button
       className={clsx(
-        'font-klima text-lg font-medium',
-        'rounded border-2 border-solid border-clementine bg-white',
+        ' rounded border-2 border-solid border-clementine bg-white',
```
Comment (zorazrr) [inline src/components/Buttons/LoadMoreButton.tsx:13]:
```suggestion
        'rounded border-2 border-solid border-clementine bg-white',
```

File: src/components/Table/AdminListTable.tsx [hunk 9]
```diff
@@ -6,7 +6,7 @@ import { TrashIcon, ArrowUturnLeftIcon } from '@heroicons/react/24/outline'
 
 const delIconStyle = clsx('ml-8 h-auto w-6 stroke-2')
 const delUserTextStyle = (deletePressed: boolean) =>
-  clsx('basis-1/3 truncate text-xl font-medium', {
+  clsx('basis-1/3 truncate ', {
```
Comment (zorazrr) [inline src/components/Table/AdminListTable.tsx:9]:
```suggestion
  clsx('basis-1/3 truncate', {
```

File: src/components/Selects/Select.tsx [hunk 10]
```diff
@@ -298,16 +298,14 @@ export const Select: FC<SelectProps> = ({
               <ul className="m-auto flex w-11/12 flex-col gap-5">
                 {Object.entries(options).map(([key, value]) => (
                   <li key={key} className="text-black">
-                    <span className="text-xl">{key}</span>
+                    <span className="">{key}</span>
```
Comment (zorazrr) [inline src/components/Selects/Select.tsx:301]:
```suggestion
                    <span>{key}</span>
```

File: src/components/Selects/Select.tsx [hunk 11]
```diff
@@ -219,7 +219,7 @@ export const Select: FC<SelectProps> = ({
         >
           <span
             className={clsx('w-full text-black', {
-              'text-md overflow-hidden whitespace-nowrap font-inter': isFilter,
+              ' overflow-hidden whitespace-nowrap': isFilter,
```
Comment (zorazrr) [inline src/components/Selects/Select.tsx:222]:
```suggestion
              'overflow-hidden whitespace-nowrap': isFilter,
```

File: src/components/Selects/Select.tsx [hunk 12]
```diff
@@ -203,7 +203,7 @@ export const Select: FC<SelectProps> = ({
       <button
         onClick={() => setIsOpen(!isOpen)}
         className={clsx(
-          'flex w-full items-center overflow-hidden rounded-full border border-solid',
+          ' flex w-full items-center overflow-hidden rounded-full border border-solid',
```
Comment (zorazrr) [inline src/components/Selects/Select.tsx:206]:
```suggestion
          'flex w-full items-center overflow-hidden rounded-full border border-solid',
```

File: src/components/Nav/SecondaryNavBar.tsx [hunk 13]
```diff
@@ -23,7 +23,7 @@ const SecondaryNavBar: FC<{ navItems: NavItem[] }> = ({ navItems }) => {
                 spy={true}
                 smooth={true}
                 duration={500}
-                className={navStyle}
+                className={clsx(navStyle, '')}
```
Comment (zorazrr) [inline src/components/Nav/SecondaryNavBar.tsx]:
```suggestion
                className={navStyle}
```

File: src/components/Text/ShadowTitle.tsx [hunk 14]
```diff
@@ -9,14 +9,14 @@ const ShadowTitle: FC<TitleProps> = ({ text }) => {
   return (
     <div
       className={clsx(
-        'w-fit rounded-full bg-white font-medium ',
+        ' header-3 w-fit rounded-full bg-white',
```
Comment (zorazrr) [inline src/components/Text/ShadowTitle.tsx:12]:
```suggestion
        'header-3 w-fit rounded-full bg-white',
```

File: src/components/Text/ShadowTitle.tsx [hunk 15]
```diff
@@ -9,14 +9,14 @@ const ShadowTitle: FC<TitleProps> = ({ text }) => {
...
         'border-4 border-cobalt',
         'px-14 py-4',
         'shadow-[-8px_8px_0px_0px] shadow-cobalt',
         'first-line:z-10',
       )}
     >
-      <p className="text-[28px] text-darkTeal ">{text}</p>
+      <p className="text-darkTeal ">{text}</p>
```
Comment (zorazrr) [inline src/components/Text/ShadowTitle.tsx:19]:
```suggestion
      <p className="text-darkTeal">{text}</p>
```

File: src/components/Text/Tag.tsx [hunk 16]
```diff
@@ -10,8 +10,8 @@ const Tag: FC<TagProps> = ({ title, className }) => {
   return (
     <div
       className={clsx(
-        'flex max-h-8 w-fit justify-center rounded-full px-[18px]',
-        'truncate text-lg font-medium',
+        'flex max-h-8 w-fit items-center justify-center rounded-full px-[18px]',
+        ' truncate',
```
Comment (zorazrr) [inline src/components/Text/Tag.tsx:14]: Merge 2 lines together!

File: src/components/Tooltips/ToolTip.tsx [hunk 17]
```diff
@@ -40,26 +40,28 @@ const ToolTip: FC<TitleProps> = ({ title, details, children }) => {
...
             )}
           />
           {children ? (
             children
           ) : (
             <>
-              <div className="font-semibold">{title}:</div>
-              {details}
+              <div className="body-small">
+                <p className="font-bold">{title}:</p>
```
Comment (zorazrr) [inline src/components/Tooltips/ToolTip.tsx:62]: Title should be bolded.

**Review by zorazrr — CHANGES_REQUESTED** [review]

**Pull request merged**